frontend/src/metabase/dashboard/components/AutoApplyFiltersToast/AutoApplyFiltersToast.tsx

```diff
@@ -74,8 +79,22 @@ export default function AutoApplyFilterToast() {
           ),
         }),
       );
+
+      return () => {
+        dispatch(dismissAutoApplyFiltersToast());
+        dispatch(dismissUndo(autoApplyFiltersToastId, false));
```

Comment:
the second argument track undo event in our Google Analytics, but this isn't actually an undo event, but a more generic toast, I think not tracking it as undo would be ideal.